### DISCUSSION TITLE
Image with architecture ARM

### DISCUSSION THREAD

**Discussion by bmgante** [backube/snapscheduler#533] [Q&A]
Hi,

Is it planned to have an image compatible with ARM architecture?

Thanks

**Comment by JohnStrunk** [comment]
While it should be possible to build for ARM, I don't have a CI infrastructure to test the resulting images. I'm hopeful that GitHub will roll out their ARM workers for general use. At that time, I'd be happy to publish multi-arch builds.

**Reply by bmgante** [reply]
@JohnStrunk could you publish an ARM image so i can test it in my eks cluster which only have arm nodes?

**Reply by ashwani77** [reply]
Hi Users, I know this is a old discussion, but do we already have any solution for this issue ?
